app/code/Magento/Bundle/Test/Unit/Model/LinkManagementTest.php

```diff
@@ -740,7 +759,8 @@ public function testSaveChild()
                     'setSelectionPriceType',
                     'setSelectionPriceValue',
                     'setSelectionCanChangeQty',
-                    'setIsDefault'
+                    'setIsDefault',
+                    'setWebsiteId'
```

Comment:
This method doesn't exist anymore on the Link, please remove it.

Reply:
Done